src/Common/DateLUTImpl.cpp

```diff
@@ -103,38 +141,14 @@ DateLUTImpl::DateLUTImpl(const std::string & time_zone_)
         else
             values.days_in_month = i != 0 ? lut[i - 1].days_in_month : 31;
 
-        values.time_at_offset_change_value = 0;
-        values.amount_of_offset_change_value = 0;
-
         if (offset_is_whole_number_of_hours_during_epoch && start_of_day > 0 && start_of_day % 3600)
             offset_is_whole_number_of_hours_during_epoch = false;
 
         if (offset_is_whole_number_of_minutes_during_epoch && start_of_day > 0 && start_of_day % 60)
             offset_is_whole_number_of_minutes_during_epoch = false;
 
-        /// If UTC offset was changed this day.
-        /// Change in time zone without transition is possible, e.g. Moscow 1991 Sun, 31 Mar, 02:00 MSK to EEST
-        cctz::time_zone::civil_transition transition{};
-        if (cctz_time_zone.next_transition(start_of_day_time_point - std::chrono::seconds(1), &transition)
-            && (cctz::civil_day(transition.from) == date || cctz::civil_day(transition.to) == date)
-            && transition.from != transition.to)
-        {
-            values.time_at_offset_change_value = (transition.from - cctz::civil_second(date)) / Values::OffsetChangeFactor;
-            values.amount_of_offset_change_value = (transition.to - transition.from) / Values::OffsetChangeFactor;
-
-            /// We don't support too large changes.
-            if (values.amount_of_offset_change_value > 24 * 4)
-                values.amount_of_offset_change_value = 24 * 4;
-            else if (values.amount_of_offset_change_value < -24 * 4)
-                values.amount_of_offset_change_value = -24 * 4;
-
-            /// We don't support cases when time change results in switching to previous day.
-            /// Shift the point of time change later.
-            if (values.time_at_offset_change_value + values.amount_of_offset_change_value < 0)
-                values.time_at_offset_change_value = -values.amount_of_offset_change_value;
-        }
-
         /// Going to next day.
+        start_of_day_time_point += std::chrono::hours(24);
```

Comment:
I've got the idea, but this variable name is weird. Let's rename it to "start_of_day_time_point_if_no_transitions" or "start_of_day_time_point_usually" or similar.

Reply:
+ writing a small comment will help.

Reply:
done